conceptual/Npgsql/compatibility.md

```diff
@@ -14,7 +14,9 @@ Please let us know if you notice any non-standard behavior.
 
 ## .NET Framework/.NET Core/mono
 
-Npgsql currently targets .NET Framework 4.6.1, as well as the [.NET Standard 2.0](https://docs.microsoft.com/en-us/dotnet/standard/net-standard) which allows it to run on .NET Core. It is also tested and runs well on mono.
+Npgsql 4.* targets .NET Framework 4.6.1, as well as the [.NET Standard 2.0](https://docs.microsoft.com/en-us/dotnet/standard/net-standard) which allows it to run on .NET Core. It is also tested and runs well on mono.
```

Comment:
```suggestion
Npgsql 4.* targets .NET Framework 4.6.1, as well as [.NET Standard 2.0](https://docs.microsoft.com/en-us/dotnet/standard/net-standard) which allows it to run on .NET Core. It is also tested and runs well on mono.
```

Npgsql 4.x also targets netstandard2.1 and netcoreapp3.1 - I'm mentioning it since for 5.* below you list all the TFMs.

I don't really think people should care about all the TFMs - only the minimum one (the others are generally to enable using specific APIs in later TFMs).

Also, I'd mention that Npgsql 4.x has a GAC installer, whereas 5.x does not.

Reply:
Added a note about GAC installer for 5.x, and the main info should be on the 'installation' page.